test/jdk/tools/jar/ExtractFilesTest.java

```diff
@@ -175,10 +175,14 @@ public void testWarningOnInvalidKeepOption() throws IOException {
                 "testfile1" + nl +
                 "testfile2" + nl;
 
-        Assertions.assertArrayEquals(baos.toByteArray(), output.getBytes());
+        assertOutputContains(output);
         Assertions.assertEquals("Warning: The --keep-old-files/-k/k option is not valid with current usage, will be ignored." + nl, err);
     }
 
+    private void assertOutputContains(String expected) {
+        Assertions.assertTrue(baos.toString().contains(expected));
+    }
+
```

Comment:
Is this really needed  vs just including the Assertions.assertTrue where you are replacing the existing Assertions.assertArrayEquals?

I only ask as there is no additional validation in the method being added so outside of  the change in methods.

Reply:
Valid points.

A separate ticker is certainly better, or perhaps I should change the ticket title. In a sense, I consider the man page change a missing part, and follow up work of the origin ticket, just like test failure.

The extra method is so that we if we need to change the verification again. The method could be better names as `verifyOutput` or else to reveal the intention better.

Reply:
> Valid points.
> 
> A separate ticket is certainly better, or perhaps I should change the ticket title. In a sense, I consider the man page change a missing part,

Yes it could have been part of the original PR, but it is separate from fixing a test bug which is why IMHO it should be a separate PR
 and follow up work of the origin ticket, just like test failure.
> 
> The extra method is so that we if we need to change the verification again. The method could be better named as `verifyOutput` or else to reveal the intention better.

If you want to keep it as is, so be it, but on the flip side, it really is not adding value currently and could just be as easily added at the point in time if/when more validation is required.

Again, just a passing observation for consideration.